pkg/api/aim/query/clause.go

```diff
@@ -19,10 +18,10 @@ func (regexp Regexp) Build(builder clause.Builder) {
 	builder.WriteQuoted(regexp.Column)
 	operator := ""
 	switch regexp.Dialector {
-	case sqlite.Dialector{}.Name():
-		operator = "regexp"
 	case postgres.Dialector{}.Name():
 		operator = "~"
+	default:
+		operator = "regexp"
 	}
 	builder.WriteString(fmt.Sprintf(" %s ", operator))
```

Comment:
for consistency, can you please use the same style between `Build` and `NegationBuild`?

Reply:
yep. done.!